### PR TITLE
UI draft and demo server added

### PR DESCRIPTION
**Pull request by Titpost**
a temporal database is used here, we will replace it later with a real one

### CODE REVIEW

File: src/main/resources/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		<h1>{{name}}</h1>
+		<div id="header"></div>
+			<div class="col-sm-10">
+				<table id="grid-table"></table>
+				<div id="grid-pager"></div>
+			</div>
+    </div>
+</body>
+
+
```
Comment (ghost) [inline src/main/resources/templates/index.html]: Format & ident, please

File: src/main/java/com/epam/lab/payments/model/ApiController.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api")
+public class ApiController {
+
+    private final CardRepository cardRepository;
+
+    public ApiController(CardRepository visitsRepository) {
```
Comment (ghost) [inline src/main/java/com/epam/lab/payments/model/ApiController.java]: Don't implement constructors by yourself, use lombok.

File: src/main/java/com/epam/lab/payments/model/ApiController.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api")
+public class ApiController {
+
+    private final CardRepository cardRepository;
```
Comment (ghost) [inline src/main/java/com/epam/lab/payments/model/ApiController.java:11]: It's a bad practice - inject repository in controller. 
Right way 
Controller -> Service -> Repository

Service - wrapper around repository or Dao layer, u should google it. For example - https://softwareengineering.stackexchange.com/questions/220909/service-layer-vs-dao-why-both

File: src/main/java/com/epam/lab/payments/model/AccountRepository.java [hunk 4]
```diff
@@ -11,6 +11,6 @@
  * Created by Александр on 03.11.2017.
  */
```
Comment (ghost) [inline src/main/java/com/epam/lab/payments/model/AccountRepository.java]: Delete signature. It's useless

File: src/main/java/com/epam/lab/payments/model/AccountRepository.java [hunk 5]
```diff
@@ -11,6 +11,6 @@
...
 public interface AccountRepository extends Repository<Account, Long> {
-    @Query("SELECT p FROM Person p WHERE LOWER(p.lastName) = LOWER(:lastName)")
-    List<Account> find(@Param("user_id") Integer user_id);
+//    @Query("SELECT p FROM Person p WHERE LOWER(p.lastName) = LOWER(:lastName)")
+//    List<Account> find(@Param("user_id") Integer user_id);
 }
```
Comment (ghost) [inline src/main/java/com/epam/lab/payments/model/AccountRepository.java]: Commented code in git - bad practice
Why simple Repository? Look at JpaRepository or CRUD repository. It contains several functions, which you need. 
And, i think u need to read little bit more about Spring Data. This query seems very strange. 
You only need to correctly write method name and params. And that's all.
BTW, find by last name? WHAT?
https://www.petrikainulainen.net/programming/spring-framework/spring-data-jpa-tutorial-three-custom-queries-with-query-methods/

File: src/main/java/com/epam/lab/payments/model/Card.java [hunk 6]
```diff
@@ -16,9 +16,10 @@
...
-    private Long id;
+    public Long id;
 
-    private String cardholderName;
-    private LocalDate expiration;
-    private Integer securityCode;
+    public String cardHolderName;
+    public LocalDate expiration;
+    public Integer securityCode;
+    public Long accountId;
```
Comment (ghost) [inline src/main/java/com/epam/lab/payments/model/Card.java:24]: Why reference type?

File: src/main/java/com/epam/lab/payments/model/IndexController.java [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+    private final CardRepository cardRepository;
+
+    public IndexController(CardRepository cardRepository) {
+        this.cardRepository = cardRepository;
+    }
+
+    @GetMapping("/")
+    public ModelAndView index() {
+        Map<String, String> model = new HashMap<>();
```
Comment (ghost) [inline src/main/java/com/epam/lab/payments/model/IndexController.java:21]: I don't understand, why it's here?

File: src/main/resources/templates/cards.css [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+			position: absolute;
+			right: 0;
+			left: 0;
+			text-align: center;
+		}
+		.ui-jqgrid .ui-jqgrid-pager .navtable {
+			height: auto;
+		}
+		}.dd
+
```
Comment (ghost) [inline src/main/resources/templates/cards.css]: This red icon - not very good.

File: src/main/java/com/epam/lab/payments/model/Card.java (hunk 6 above)
Comment (Astel) [reply src/main/java/com/epam/lab/payments/model/Card.java:24]: JPA can work with LocalDate, isn't it? And there is reference type in all guide.

File: src/main/java/com/epam/lab/payments/model/AccountRepository.java (hunk 5 above)
Comment (Astel) [reply src/main/java/com/epam/lab/payments/model/AccountRepository.java]: lastname is my bad for pushing template of query annotation

File: src/main/java/com/epam/lab/payments/model/IndexController.java (hunk 7 above)
Comment (Titpost) [reply src/main/java/com/epam/lab/payments/model/IndexController.java:21]: This is temporal DB stub, we delete it later. Real DB is already created and will replace all this junk very soon

File: src/main/java/com/epam/lab/payments/model/ApiController.java (hunk 3 above)
Comment (Titpost) [reply src/main/java/com/epam/lab/payments/model/ApiController.java:11]: This is also temporal staff, just for quickness. Will be replaced later with a real REST service

File: src/main/java/com/epam/lab/payments/model/ApiController.java (hunk 2 above)
Comment (Titpost) [reply src/main/java/com/epam/lab/payments/model/ApiController.java]: This file will be deleted and replaced with another controller.

File: src/main/java/com/epam/lab/payments/model/IndexController.java [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+        model.put("name", "Cards:");
+
+        Card card = new Card();
+        card.cardHolderName = "IVANOV STEPAN";
+        card.securityCode = 777;
+        card.accountId = new Long(12345678);
```
Comment (iliaonishchenko) [inline src/main/java/com/epam/lab/payments/model/IndexController.java:27]: It is not java-way for field initialization.
You are using lombok, so use constructor (`@AllArgsConstructor`) or builder pattern (`@Builder`)
Also, unnecessaary boxing here.

**Review by ghost — APPROVED** [review]

**Pull request merged**